### PR TITLE
chore: 2.1.1 release

### PR DESCRIPTION
**Pull request by tylerbrandt**

### CODE REVIEW

File: packages/optimizely-sdk/CHANGELOG.MD [hunk 1]
```diff
@@ -1,3 +1,8 @@
+## 2.1.1
+June 19, 2018
+
+* Fix: send impression event for Feature Test with Feature disabled ([#117](https://github.com/optimizely/javascript-sdk/pull/117))
```
Comment (aliabbasrizvi) [inline packages/optimizely-sdk/CHANGELOG.MD]: nit. Period at the end of line.

Comment (aliabbasrizvi) [reply packages/optimizely-sdk/CHANGELOG.MD]: And perhaps `...when Feature is disabled`?

Comment (tylerbrandt) [reply packages/optimizely-sdk/CHANGELOG.MD]: We don't use periods elsewhere in the changelog

Comment (tylerbrandt) [reply packages/optimizely-sdk/CHANGELOG.MD]: at least not consistently

**Pull request merged**